lsp-go.el

```diff
@@ -78,11 +80,32 @@ completing function calls."
   :risky t
   :package-version '(lsp-mode "6.2"))
 
+(defvar lsp-gopls-available-codelens
+  '(("generate" . "Run `go generate` for a directory")
+	("test" . "Run `go test` for a specific test function")
+	("tidy" . "Run `go mod tidy` for a module")
+	("upgrade_dependency" . "Upgrade a dependency")
+	("regenerate_cgo" . "Regenerate cgo definitions"))
+  "Available codelens that can be further enabled or disabled
+  through `lsp-gopls-codelens'.")
+
+(defcustom lsp-gopls-codelens '(("generate" . t) ("test" . t))
+  "Select what codelens should be enabled or not.
+
+The codelens can be found at https://github.com/golang/tools/blob/4d5ea46c79fe3bbb57dd00de9c167e93d94f4710/internal/lsp/source/options.go#L102-L108."
+  :type (lsp--defcustom-available-as-alist-type lsp-gopls-available-codelens)
+  :group 'lsp-gopls
+  :set 'lsp--defcustom-set-json-alist
```

Comment:
we should not use :get and :set since they are not supported in emacs 25. I propose removing them and ask users to enter :json-false when they need false.

Reply:
Ah sorry @yyoncho yes I read that but forgot when I implemented.
And in fact, I believe I can just change https://github.com/emacs-lsp/lsp-mode/pull/1766/files#diff-b1e9bc82c3b6bd6e95470e18ee41a19bR8460 to set `:json-false` instead of `nil`.